pyproject.toml

```
@@ -0,0 +1,139 @@
+[build-system]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sdm_tools"
```

```suggestion
name = "lsst_sdm_tools"
```
The name is always the full name. (and if you look in your python directory you will see that the .dist-info directory picks up this name as the distribution name. Be consistent and include the `lsst` prefix.